english/marshmallow.md

```diff
@@ -1,15 +1,15 @@
 Once upon a time many authors collaborated to make meaningless linked story.
 
-[start](mad/mad.md) your journey into world of mad 
+[start](mad/mad.md) your journey into world of mad [trolls](overwatch/overwatch.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.